Cargo.toml

```diff
@@ -2,9 +2,11 @@
 members = [
   "crates/codspeed",
   "crates/bencher_compat",
-  "crates/criterion_compat",
   "crates/cargo-codspeed",
 
+  "crates/criterion_compat",
+  "crates/criterion_compat/criterion-0.5.1",
```

Comment:
This folder named with the version can be misleading since it is another (forked version). We should keep the same format we had for the divan fork